src/components/Records/Record/GeneralInfo.vue

```diff
@@ -38,16 +38,25 @@
         <div class="d-flex flex-column mt-2  ml-sm-6 ml-lg-8">
           <div class="d-flex flex-row mb-2 align-center">
             <h3>{{ currentRecord['fairsharingRecord'].name }}</h3>
-            <b class="ml-2">({{ currentRecord['fairsharingRecord'].abbreviation
-            }})</b>
+            <b
+              v-if="currentRecord['fairsharingRecord'].abbreviation"
+              class="ml-2"
+            >({{ currentRecord['fairsharingRecord'].abbreviation }})</b>
           </div>
           <div class="d-flex align-center">
             <h3 class="mr-1">
               doi:
             </h3>
-            <a :href="currentRecord['fairsharingRecord'].doi">
+            <a
+              v-if="currentRecord['fairsharingRecord'].doi"
+              :href="currentRecord['fairsharingRecord'].doi"
```

Comment:
this need to have target="_blank" as the DOI will very likely be an external link